src/interface/mesh_data.hpp

```diff
@@ -88,7 +89,7 @@ class MeshData {
 
   void SetAllowedDt(const Real dt) const {
```

Comment:
I feel like the name of this method needs to be changed because it is no longer just a set. E.g. ConsolidateAllowedDt though even that is not entirely clear either.

Reply:
I agree (and would even go further as some naming doesn't seem to match the implementation any more).
Should I fix this here or in a separate PR?

Reply:
I'm fine if you merge and open a WIP pr to fix it.

Reply:
Sounds good. I'll merge then (one thing off the table).